docs/reference/indices/rollover-index.asciidoc

```diff
@@ -24,17 +56,102 @@ from the original (rolled-over) index.
 In this scenario, the write index will have its rollover alias' `is_write_index` set to `false`, while the newly created index
 will now have the rollover alias pointing to it as the write index with `is_write_index` as `true`.
 
-The available conditions are:
 
-[[index-rollover-conditions]]
-.`conditions` parameters
-[options="header"]
-|===
-| Name     | Description
-| max_age  | The maximum age of the index
-| max_docs | The maximum number of documents the index should contain. This does not add documents multiple times for replicas
-| max_size | The maximum estimated size of the primary shard of the index
-|===
+[[rollover-wait-active-shards]]
+===== Wait for active shards
+
+Because the rollover operation creates a new index to rollover to, the
+<<create-index-wait-for-active-shards,`wait_for_active_shards`>> setting on
+index creation applies to the rollover action as well.
+
+
+[[rollover-index-api-path-params]]
+==== {api-path-parms-title}
+
+`<alias>`::
+(Required, string)
+Name of the existing index alias
+to assign to the target index.
+
+
+`<target-index>`::
++
+--
+(Optional*, string)
+Name of the target index to create and assign the index alias.
+
+include::{docdir}/indices/create-index.asciidoc[tag=index-name-reqs]
+
+*This parameter is not required
+if the alias is assigned to an index name that ends with `-` and a number,
+such as `logs-000001`.
+In this case,
+the name of the new index follows the same pattern,
+incrementing the number.
+For example,
+`logs-000001` increments to `logs-000002`.
+This number is zero-padded with a length of 6,
+regardless of the prior index name.
+
+If the existing index for the alias does not match this pattern,
+this parameter is required.
+--
+
+
+[[rollover-index-api-query-params]]
+==== {api-query-parms-title}
+
+`dry_run`::
+(Optional, boolean)
+If `true`,
+the request checks whether the index matches provided conditions
+but does not perform a rollover.
+Defaults to `false`.
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=include-type-name]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=wait_for_active_shards]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=timeoutparms]
+
+
+[[rollover-index-api-request-body]]
+==== {api-request-body-title}
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=aliases]
+
+`conditions`::
++
+--
+(Required, object)
+Set of conditions the index alias's existing index must met to rollover.
+
```

Comment:
I'd say "to roll over" rather than close it up. (Another nit.)

Reply:
Fixed with b39cfcc27b78c0bbf1a2a1f4cd5bc1a78459dc60